example3.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+
+import logging
+
+from logstash_async.handler import AsynchronousLogstashHandler
+
+host = 'localhost'
+port = 5959
+
+logger = logging.getLogger('logstash_async.transport')
+logger.setLevel(logging.DEBUG)
+handler = logging.StreamHandler()
+logging_format = '%(asctime)s.%(msecs)03d'
+logging_format += ' | %(levelname)-8s'
+logging_format += ' | %(name)s.%(module)s.%(funcName)s'
+logging_format += ' | %(message)s'
+date_format = '%Y-%m-%d | %H:%M:%S'
+formatter = logging.Formatter(logging_format, date_format)
+handler.setFormatter(formatter)
+logger.addHandler(handler)
```

Comment:
`logger` and so also `handler` and `formatter` are not used at all, so can be removed.

Reply:
IThis is the configuration for the logstash_async.transport logger. To see the debugging logs from there.

Reply:
Ah, sorry.